### ISSUE TITLE
Bad color for the h1 of the index

### ISSUE THREAD

**Issue by jibe0123** [IW4-GROUPE-AMG/Unsecure#7]
**Description**  
We can't really read the title 

**How to reproduce**  
Browse /

**Possible Solution**  
change the css 
`    color: black;
    font-weight: bold;
    font-size: 44pt;
    text-align: center;
    background-color: white;`

**Additional context**

**Comment by walidmh** [comment]
I think that the color change are good to do , but the white background color are not too intersting because it will be hiding the picture of the city. What do you think @Olivier1234 ?

**Comment by Olivier1234** [comment]
Yes, besides the white the colors looks good. And it would be good to add another font family too.
I think COMICS 100 ms would be good.
